heaps/min_heap.py

```diff
@@ -21,21 +21,33 @@ def __init__(self):
     def add(self, key, value = None):
```

Comment:
👍  However space complexity is O(1) since you just add 1 element to the heap and use an iterative solution